README.md

```diff
@@ -14,7 +14,7 @@ The 1Password Connect collection contains modules that interact with your 1Passw
 
 ## Requirements
 - `ansible`: **>=7.x**
-- `ansible-core`: **>=2.14**
+- `ansible-core`: **>=2.15**
 - `python`: **>=3.8**
```

Comment:
```suggestion
- `python`: **>=3.9**
```

Or, honestly, since we aren't (yet) testing against python 3.12

```suggestion
- `python`: >=3.9, <=3.11**
```